R/bundle.R

```diff
@@ -120,6 +120,46 @@ isWindows <- function() {
   Sys.info()[["sysname"]] == "Windows"
 }
 
+versionFromDescription <- function(appDir) {
+  descriptionFilepath <- file.path(appDir, "DESCRIPTION")
+  if (!file.exists(descriptionFilepath)) {
+    return(NULL)
+  }
+
+  desc <- read.dcf(descriptionFilepath)
+  depends <- as.list(desc[1, ])$Depends
+  if (is.null(depends)) {
+    return(NULL)
+  }
+
+  regexExtract("R \\((.*?)\\)", depends)
+}
+
+versionFromLockfile <- function(appDir) {
+  tryCatch(
+    {
+      lockfile <- suppressWarnings(renv::lockfile_read(project = appDir))
+      paste(">=", lockfile$R$Version)
```

Comment:
In Python we are using `~= V` instead of `>= V` like we do here.
Should we keep the behaviour consistent or in R is more pragmatical to always ask for >= when the user specified a version?

Reply:
Ha, jynx